docs/Execution.md

```diff
@@ -135,3 +135,17 @@ Check the bundle's `stdout`, and you should see something similar to before:
     ...
 
 And that's all.  Congrats!
+
+## Frequently asked questions
+
+### Can I reserve a worker for a particular tag?
+
+Yes, you can use the `--tag-exclusive` flag of `cl worker` (along with a
+`--tag`) to prevent your worker from running untagged bundles, since
+`tag_exclusive` workers only run bundles that match their tag.
+
+To be concrete, a worker started with `cl worker --tag debug --tag-exclusive`
+will _only_ run bundles with `--request-queue tag=debug`. This is especially
```

Comment:
We're trying to deprecate `tag=`, so now you can just do `--request-queue debug` (former is still supported for now).